src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -382,27 +385,35 @@ public String call() throws RuntimeException {
             result.append(min).append(" Version: ").append(System.getProperty("java.vm.specification.version"))
                     .append("\n");
             result.append(maj).append(" JVM Implementation\n");
-            result.append(min).append(" Name:    ").append(System.getProperty("java.vm.name")).append("\n");
-            result.append(min).append(" Vendor:  ").append(System.getProperty("java.vm.vendor")).append("\n");
-            result.append(min).append(" Version: ").append(System.getProperty("java.vm.version")).append("\n");
+            result.append(min).append(" Name:    ").append(
+                    Markdown.escapeUnderscore(System.getProperty("java.vm.name"))).append("\n");
+            result.append(min).append(" Vendor:  ").append(
+                    Markdown.escapeUnderscore(System.getProperty("java.vm.vendor"))).append("\n");
```

Comment:
Because they're raw strings which could potentially have underscores in them, and thus they should be escaped.

The whole purpose of this PR is to escape strings that might have `_` or `` ` `` outside `` `...` ``.

Reply:
Then why not escaping both? I guess you checked possible names/examples and you acted accordingly, but maybe we should be more conservative?

Reply:
Conservative meaning trying to escape both characters.

Reply:
@jsoref did you have the opportunity to think about my comment?

Reply:
I've seen underscores in various parts of Java things, so being somewhat conservative about that character seems worthwhile. I've found a list of vendors, but I haven't found any actual lists of `java.vm.name` / `java.vm.vendor` values ☹.

It seems fairly unlikely that either field will include `` ` `` which is why I didn't.

I'm also have expecting that once this code is split/refactored, I'd switch whole lines to a thing that escapes both at once...